src/items/rules/augmentations.json

```diff
@@ -0,0 +1,214 @@
+        "format": 1,
+        "markdown": ""
+      },
+      "title": {
+        "level": 1,
+        "show": true
+      },
+      "video": {
+        "controls": true,
+        "volume": 0.5
+      }
+    },
+    {
+      "_id": "2wI6ORVoljJhY21v",
+      "name": "Species Grafts",
+      "type": "text",
+      "flags": {},
+      "image": {},
+      "ownership": {
+        "default": -1
+      },
+      "sort": 700000,
+      "src": null,
+      "system": {},
+      "text": {
+        "content": "<p><strong>Source:</strong> AA4 pg. 140</p><p><span id=\"ctl00_MainContent_Header3\">This section introduces a new type of augmentation known as a species graft, giving you the ability to partially transcend your species by taking on the racial traits of other species. Species grafts follow the rules for augmentations, in that they are installed in specific systems on your body. Once installed, they become a part of your body and generally can't be affected by abilities that destroy or disable objects. Most of these augmentations are installed into the more common systems. <br /><br />Species grafts can be cybernetics, biotech, magitech, nanite enhancements, or any other form of augmentation. A specific species graft might describe its form, but that is merely an example. Your GM might decide what a species graft looks like when you purchase one or you might determine its specifics when you craft it. This doesn't affect how a species graft functions—only its appearance. <br /><br />The benefits of a species graft don't stack with the benefits granted by a racial trait of the same name. For example, a vesk character can't benefit from the armor savant graft. Species grafts are different from other augmentations in that they count as the racial trait mentioned in the name of the graft, and having one installed allows you to take feats and other player options that have that racial trait as a prerequisite. For example, if you have a limited telepathy graft installed, you count as having the limited telepathy racial trait in order to satisfy the prerequisite for the </span><span style=\"text-decoration:underline\"><a href=\"https://www.aonsrd.com/Feats.aspx?ItemName=Extended%20Telepathy\"><span style=\"text-decoration:underline\">Extended Telepathy</span></a></span> feat. You can't replace racial traits granted by species grafts with alternate racial traits, as those must be chosen at 1st level. <br /><br />If you have a species graft removed, you no longer count as having the associated racial trait. If you have feats or other options with that racial trait as a prerequisite, you can no longer use them. You can replace a feat with a new one that has no prerequisites by spending a full week retraining.</p>",
```

Comment:
Clean out the spans in this page, and replace `<br />`s with a new paragraph. Get rid of links to AoN